# Sensor data
std_msgs/Header header
string sensorDataInfo
int32 size
int32 type
uint8[] sensorData
